mmf/datasets/mmf_dataset_builder.py

```diff
@@ -119,6 +123,49 @@ def _download_requirement(
 
     def load(self, config, dataset_type, *args, **kwargs):
         self.config = config
+
+        new_config = config
+        split_dataset_from_train = self.config.split_train
```

Comment:
This will break existing datasets, use 
`self.config.get("split_train", False)`